lib/elixir/lib/kernel.ex

```diff
@@ -5205,7 +5205,7 @@ defmodule Kernel do
       ~D[2015-01-13]
 
   """
-  defmacro sigil_D(date, modifiers)
+  defmacro sigil_D(term, modifiers)
```

Comment:
I think we could go with `date_string`, `time_string`, `naive_datetime_string`, etc. Thoughts?

Reply:
Works for me 👍